src/utils/Helper.js

```diff
@@ -21,14 +54,14 @@ export default class Helper {
   }
 
   /**
-   * @method buildErrorResponse
-   * @description Set error response
+   * @method handleValidation
+   * @description Handle validation errors
    * @static
    * @param {object} data
    * @returns {object} JSON response
    * @memberof Helper
    */
-  static buildErrorResponse(data) {
+  static handleValidation(data) {
```

Comment:
This method's name does not need to change. Please revert it.

Reply:
Changes implemented